### PR TITLE
Add Contract Address SyncDEX Finance

### PR DESCRIPTION
**Pull request by devsyncdex**
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/syncdex-finance: 
        
 
  
    
    
    Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (transaction={"to":"0x1A15e21cc6C26d48EE79DC45F704E9593ffbA81A","data":"0x574f2ba3","accessList":null}, code=CALL_EXCEPTION, version=providers/5.7.2)
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/providers/lib/fallback-provider.js:654:52
        at Array.forEach (<anonymous>)

**Comment by llamatester** [comment]
Error while running adapter at projects/pancakeswap-v3: 
        
 
  
    
    
    Error: failed to meet quorum (method="getLogs", params={"filter":{"fromBlock":"0x19c9236","toBlock":"0x19c9260","address":"0x0BFbCF9fa4f9C56B0F40a671Ad40E0805A091865","topics":["0x783cca1c0412dd0d695e784568c96da2e9c22ff989357a2e8b1d9b2b4e6b7118"]}}, results=[{"weight":1,"start":1680558514548,"error":{"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":46,\"error\":{\"code\":-32005,\"message\":\"limit exceeded\"}}","error":{"code":-32005},"requestBody":"{\"method\":\"eth_getLogs\",\"params\":[{\"fromBlock\":\"0x19c9236\",\"toBlock\":\"0x19c9260\",\"address\":\"0x0bfbcf9fa4f9c56b0f40a671ad40e0805a091865\",\"topics\":[\"0x783cca1c0412dd0d695e784568c96da2e9c22ff989357a2e8b1d9b2b4e6b7118\"]}],\"id\":46,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://bsc-dataseed.binance.org/"}},{"weight":1,"start":1680558515298,"error":{"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":43,\"error\":{\"code\":-32005,\"message\":\"limit exceeded\"}}","error":{"code":-32005},"requestBody":"{\"method\":\"eth_getLogs\",\"params\":[{\"fromBlock\":\"0x19c9236\",\"toBlock\":\"0x19c9260\",\"address\":\"0x0bfbcf9fa4f9c56b0f40a671ad40e0805a091865\",\"topics\":[\"0x783cca1c0412dd0d695e784568c96da2e9c22ff989357a2e8b1d9b2b4e6b7118\"]}],\"id\":43,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://bsc-dataseed1.defibit.io/"}},{"weight":1,"start":1680558516048,"error":{"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":43,\"error\":{\"code\":-32005,\"message\":\"limit exceeded\"}}","error":{"code":-32005},"requestBody":"{\"method\":\"eth_getLogs\",\"params\":[{\"fromBlock\":\"0x19c9236\",\"toBlock\":\"0x19c9260\",\"address\":\"0x0bfbcf9fa4f9c56b0f40a671ad40e0805a091865\",\"topics\":[\"0x783cca1c0412dd0d695e784568c96da2e9c22ff989357a2e8b1d9b2b4e6b7118\"]}],\"id\":43,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://bsc-dataseed1.ninicoin.io/"}},{"weight":1,"start":1680558516798,"error":{"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":43,\"error\":{\"code\":-32005,\"message\":\"limit exceeded\"}}","error":{"code":-32005},"requestBody":"{\"method\":\"eth_getLogs\",\"params\":[{\"fromBlock\":\"0x19c9236\",\"toBlock\":\"0x19c9260\",\"address\":\"0x0bfbcf9fa4f9c56b0f40a671ad40e0805a091865\",\"topics\":[\"0x783cca1c0412dd0d695e784568c96da2e9c22ff989357a2e8b1d9b2b4e6b7118\"]}],\"id\":43,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://bsc-dataseed2.defibit.io/"}},{"weight":1,"start":1680558517548,"error":{"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":43,\"error\":{\"code\":-32005,\"message\":\"limit exceeded\"}}","error":{"code":-32005},"requestBody":"{\"method\":\"eth_getLogs\",\"params\":[{\"fromBlock\":\"0x19c9236\",\"toBlock\":\"0x19c9260\",\"address\":\"0x0bfbcf9fa4f9c56b0f40a671ad40e0805a091865\",\"topics\":[\"0x783cca1c0412dd0d695e784568c96da2e9c22ff989357a2e8b1d9b2b4e6b7118\"]}],\"id\":43,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://bsc-dataseed2.ninicoin.io/"}}], provider="[object Object]", code=SERVER_ERROR, version=providers/5.7.2)
    Truncated error stack:
        at FallbackProvider.<anonymous> (/home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/@ethersproject/providers/lib/fallback-provider.js:684:54)

File: projects/syncdex-finance/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const { getUniTVL } = require('../helper/unknownTokens')
+
+module.exports = {
+  misrepresentedTokens: true,
+  era: {
+    tvl: getUniTVL({
+      factory: '0x1A15e21cc6C26d48EE79DC45F704E9593ffbA81A',
```
Comment (g1nt0ki) [inline projects/syncdex-finance/index.js:7]: this does not seem to be uniswap factory address, more like presale contract?

**Comment by devsyncdex** [comment]
Sorry for the confusion. I have pushed the new code and would appreciate it if you could check it.

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
@devsyncdex sorry, we no longer accept fetch adapters, please provide factory details, I will list the project

**Comment by devsyncdex** [comment]
The purpose of the project will be shifted to focus on staking and farming, therefore, the TVL will be rebuilt by counting the assets in the staking contract. Please check it, have a nice day team.

File: projects/syncdex-finance/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const sdk = require("@defillama/sdk");
+
+const staking_contract = "0xa8CD01322Ad632c9656879e99Fd7FbC11ca8E3BB";
```
Comment (g1nt0ki) [inline projects/syncdex-finance/index.js:3]: I dont see any tokens here

Comment (g1nt0ki) [reply projects/syncdex-finance/index.js:3]: https://zksync2-mainnet.zkscan.io/address/0xa8CD01322Ad632c9656879e99Fd7FbC11ca8E3BB/coin-balances

File: projects/syncdex-finance/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+const assets = [
+  // other tokens which probably for some reason was sent to the contract accidentally
+  "0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4",
+  "0x3a34FA9a1288597Ad6C1Da709f001D37FeF8b19e",
```
Comment (g1nt0ki) [inline projects/syncdex-finance/index.js]: sorry, you can't include your projects own tokens as tvl

**Review by g1nt0ki — CHANGES_REQUESTED** [review]
sorry, we cant list your project till there is some tvl

**Comment by llamatester** [comment]
The adapter at projects/syncdex-finance exports TVL: 
        
 
 
    era-staking               0
    era                       0
    staking                   0
    
    total                    0

File: projects/syncdex-finance/index.js (hunk 3 above)
Comment (devsyncdex) [reply projects/syncdex-finance/index.js]: > sorry, we cant list your project till there is some tvl
Hi team, the contract has just been added with some TVL and will be gradually migrated to it in the next few days. Please check it and list the projects to help Era's ecosystem grow stronger. Have a nice day team!

**Comment by devsyncdex** [comment]
Dear team, the contract has just been added with some TVL and will be gradually migrated to it in the next few days. Please check it and list the projects to help Era's ecosystem grow stronger. Have a nice day team!

**Comment by devsyncdex** [comment]
> sorry, we cant list your project till there is some tvl



> The adapter at projects/syncdex-finance exports TVL:
> 
> ```
> era-staking               0
> era                       0
> staking                   0
> 
> total                    0 
> ```

Hello there, please take a moment to check out my latest update. Thank you sincerely.

**Pull request closed without merging**